packages/peer-exchange/src/waku_peer_exchange_discovery.ts

```diff
@@ -46,21 +57,22 @@ export class PeerExchangeDiscovery
   private readonly peerExchange: WakuPeerExchange;
   private readonly options: Options;
   private isStarted: boolean;
-  private intervals: Map<PeerId, NodeJS.Timeout> = new Map();
+  private intervals: Map<string, NodeJS.Timeout> = new Map();
```

Comment:
```suggestion
  private intervals: Map<string, number> = new Map();
```
Odd to use a NodeJS type when we target the browser.